src/99-appendices/03-hed.md

```diff
@@ -167,21 +167,71 @@ This allows for a proper validation of the HED annotations
 (for example using the `bids-validator`).
 
 Example: The following `dataset_description.json` file specifies that the
-[`HED8.0.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.0.0.xml)
+[`HED8.1.0.xml`](https://github.com/hed-standard/hed-specification/tree/master/hedxml/HED8.1.0.xml)
 file from the `hedxml` directory of the
 [`hed-specification`](https://github.com/hed-standard/hed-specification)
 repository on GitHub should be used to validate the study event annotations.
 
 ```JSON
 {
   "Name": "A great experiment",
-  "BIDSVersion": "1.6.0",
-  "HEDVersion": "8.0.0"
+  "BIDSVersion": "1.7.0",
+  "HEDVersion": "8.1.0"
 }
 ```
 
 If you omit the `HEDVersion` field from the dataset description file,
-any present HED information will be validated using the latest version of the HED schema,
-which is bound to result in problems.
-Hence, it is strongly RECOMMENDED that the `HEDVersion` field be included when using HED
-in a BIDS dataset.
+any present HED information will be validated using the latest version of the HED schema.
+This is bound to result in problems, and hence, it is strongly RECOMMENDED that the
+`HEDVersion` field be included when using HED in a BIDS dataset.
+
+### Using HED library schemas
+
+HED also allows you to use one or more specialized vocabularies along with or instead of
+the standard vocabulary. These specialized vocabularies are developed by
+communities of users and are available in the GitHub
+[hed-schema-library](https://github.com/hed-standard/hed-schema-library) repository.
+Library schema are specified by *library-name*_*library-version*.
```

Comment:
```suggestion
Library schema are specified in the form `<library-name<_>library-version>`.
```

Reply:
Library schema names must be alphabetic.  I read over the HED specification, and it is not spelled out as clearly as it should be. I'll make sure it is.  Library schema must use semantic versioning.  Again, it isn't spelled out as clearly as it could be in the HED specification. I'll make sure that it is.  Thanks for your suggestions.